docs/operations-manual.md

```diff
@@ -227,8 +219,7 @@ helm delete metadig-solr -n metadig
 
 ### DataONE token
 
-The metadig-engine uses a k8s secret to contain the CN administrative token. This token is used by metadig-scorer and metadig-scheduler services in order to read private content (sysmeta and metadata) that is necessary to create metadata
-assessments and graphics. The k8s secret is only created using a kubectl command and is never contained in any Helm chart or manifest file. This secret is made available to metadig pods as an environment variable, for example, see https://github.com/NCEAS/metadig-engine/blob/feature-helm-%23152/helm/metadig-scheduler/templates/deployment.yaml#L39
+The metadig-engine uses a k8s secret to contain the CN administrative token. This token is used by metadig-scorer and metadig-scheduler services in order to read private content (sysmeta and metadata) that is necessary to create metadata assessments and graphics. The k8s secret is only created using a kubectl command and is never contained in any Helm chart or manifest file. This secret is made available to metadig pods as an environment variable, for example, see [the scheduler deployment.yaml](https://github.com/NCEAS/metadig-engine/blob/feature-helm-%23152/helm/metadig-scheduler/templates/deployment.yaml#L39) Note that this secret only needs to be updated if the token becomes invalid (due to cert renewal, for example). A `helm upgrade` will keep the token in place.
```

Comment:
What is the name of the secret that is used?

Reply:
you mean in kubernetes? it is all shown below this snippet (which was just reformatted to remove an unnecessary line break, like some of the other changes in this doc)